### PR TITLE
Use cache for more fields of experiment run

### PR DESCRIPTION
**Pull request by conradoverta**

### CODE REVIEW

File: client/verta/verta/_tracking/experimentrun.py [hunk 1]
```diff
@@ -666,16 +670,8 @@ def get_tags(self):
...
-        msg = Message(id=self.id)
-        data = _utils.proto_to_json(msg)
-        response = _utils.make_request("GET",
-                                       "{}://{}/api/v1/modeldb/experiment-run/getExperimentRunTags".format(self._conn.scheme, self._conn.socket),
-                                       self._conn, params=data)
-        _utils.raise_for_http_error(response)
-
-        response_msg = _utils.json_to_proto(_utils.body_to_json(response), Message.Response)
-        return response_msg.tags
+        self._clear_cache()
```
Comment (convoliution) [inline client/verta/verta/_tracking/experimentrun.py:673]:
```suggestion
        self._refresh_cache()
```

Comment (convoliution) [inline client/verta/verta/_tracking/experimentrun.py:673]:
```suggestion
        self._refresh_cache()
```

**Review by convoliution — CHANGES_REQUESTED** [review]

**Comment by convoliution** [comment]
I noticed that the `get_tags()` test fails (returns fewer items than expected). Need to check if it's a quirk of the test or caching tags.

**Comment by conradoverta** [comment]
> I noticed that the `get_tags()` test fails (returns fewer items than expected). Need to check if it's a quirk of the test or caching tags.

This is weird =)

**Pull request closed without merging**